docs/simreal/simvsreal.rst

```diff
@@ -1,29 +1,16 @@
 .. _sec-simulation-vs-real-robot:
 
-**************************
-Simulation vs Real Robot
-**************************
-The ``SimFinger`` class provides us with an API to control the TriFinger robots in
-simulation. And, how can these robots be controlled in real life? via the
-`robot_interfaces::RobotFrontend`_ class.
-
-We aim to have the same API for controlling the robot in the ``SimFinger`` class
-as in ``robot_interfaces::RobotFrontend`` to allow transition from simulation to
-real robot with only minimal modifications to the code. There are still a few
-important differences one should keep in mind though. These differences are
-described below:
+********************************************************
+Switching between Non-real-time and Real-time Behaviour
+********************************************************
 
-.. note:: 
-
-  It is also possible to use our simulation through ``robot_interfaces``
-  by using a backend that uses the pyBullet driver. This way, the full
-  functionality of ``robot_interfaces`` in addition to that provided by
-  ``SimFinger`` is available (so you won't have to keep the below
-  differences in mind betwen the simulation and the real robot). 
-  However, a drawback of using this approach is that this way it is
-  not possible to directly access the simulation, e.g. to reset the finger.
-  For more details see :ref:`robot_interfaces with Simulation`.
+The ``SimFinger`` class provides us with an API to control the TriFinger robots in
+simulation. Similarly, the `robot_interfaces::RobotFrontend`_ class provides an
+API to interact with these robots in real life.
 
+It is extremely easy to switch between these two. However, there are some subtle
```

Comment:
Maybe a matter of taste but I would be a bit careful with words like "extremely".  I think "It is easy to switch" would be enough here.

Reply:
Yeah, I think it's my writing style: I tend to use more adjectives and adverbs. It is, in fact, extremely easy to use :P So we could make that explicit like this